src/Typewriter/Properties/launchSettings.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
maybe we should exclude the launch settings from the repo and provide templates to easily onboard new developers? And do the same for vscode settings
(for some reason I can't comment on the file below with the path)